<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configure</class>
 <widget class="QDialog" name="Configure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="sbIpField4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbIpField1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
   </property>
   <property name="suffix">
    <string>.</string>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConfigure">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>210</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ip:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbIpField2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
   </property>
   <property name="suffix">
    <string>.</string>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port RP:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPortRp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>20030</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port ANT:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log File Path:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPortAnt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>20080</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbIpField3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
   </property>
   <property name="suffix">
    <string>.</string>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="leConfigFilePath">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>C:/Pritesh/FCR_Controller_Sim/test.txt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBrowse">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sbIpField1</tabstop>
  <tabstop>sbIpField2</tabstop>
  <tabstop>sbIpField3</tabstop>
  <tabstop>sbIpField4</tabstop>
  <tabstop>sbPortRp</tabstop>
  <tabstop>sbPortAnt</tabstop>
  <tabstop>leConfigFilePath</tabstop>
  <tabstop>pbBrowse</tabstop>
  <tabstop>pbConfigure</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
